src/core/currency/wallet/currency-wallet-api.ts

```diff
@@ -532,6 +533,17 @@ export function makeCurrencyWalletApi(
       )
     },
 
+    async signBytes(
+      bytes: Uint8Array,
+      opts: EdgeSignBytesOptions = {}
+    ): Promise<string> {
+      if (engine.signBytes == null) {
+        throw new Error(`${pluginId} doesn't support signing bytes`)
```

Comment:
We talked about adding a bit in here, `if (pluginId == 'ethereum') return engine.signMessage(base16.stringify(bytes),...)`.

This would be an optional change, just to get quick support for this new feature without engine changes.